src/pages/loginPage/LoginPage.js

```diff
@@ -22,37 +19,30 @@ const LoginPage = () => {
   }
 
   return (
-    <div className="flex-container-centered">
-      <div className="flex-item flex-column-container left-side">
-        <div className="login-text">
-          <Smilies className="smilies" />
-          <h2 className="login_title">TARGET MVD</h2>
-          <h4>Find people near you & Connect</h4>
-          <p>
-            Create a target wherever on the map, specify your interest: Travel, Dating, Music, etc
-            and start conecting with others who share your interest.
-          </p>
-        </div>
-        <div className="login-form">
-          <LoginForm onSubmit={loginRequest} />
-          <p className="forgot-password">Forgot your password?</p>
-        </div>
-        <p className="sign-in-facebook">Connect with facebook</p>
-        <hr />
-        <NavLink to={routes.signUp} className="signup-button">
-          <FormattedMessage id="login.signup" />
-        </NavLink>
-      </div>
-
-      <div className="flex-item blue-background flex-column-container">
-        <Iphone6 className="iphone-6" />
-        <AppStoreBtn />
-        <div className="flex-container social-buttons-container">
-          <FacebookBtn />
-          <TwitterBtn />
-        </div>
-      </div>
-    </div>
+    <LandingLayout
+      leftSideElement={
+        <>
+          <div className="login-text">
+            <Smilies className="smilies" />
+            <h2 className="login_title">TARGET MVD</h2>
```

Comment:
Should this be at your i18n file? something like this:
```suggestion
            <h2 className="login_title"><FormattedMessage id="login.title" /></h2>
```